docs/manage/users.rst

```diff
@@ -58,6 +61,17 @@ are known as remote users.
 
 To find out more, visit :ref:`remote-users`.
 
+Password Requirements
+=====================
+
+Passwords:
+
+-  cannot be blank
+-  must be at least 8 characters
+-  must contain at least one upper-case letter
+-  must contain at least one lower-case letter
+-  must contain at least one number
```

Comment:
Are these actually current requirements, or just suggestions?  I have pretty strong opinions on password complexity requirements, and they closely align to [NIST 800-63b](https://pages.nist.gov/800-63-3/sp800-63b.html), section 5.1.1. :)
cc: @ioga

Reply:
Since the requirements haven't been implemented yet, we should probably leave that section out of the docs.  Or reword it as a suggestion like "Organizations typically have local password requirements.  In the absence of local guidance, a strong password is typically a multi-word phrase longer than than 8 characters, does not contain a sequence like abc123, and is not a single dictionary word.  For example, a short sentence like `I love using Determined.ai` is easy to remember but hard to guess."

Reply:
blocked until https://github.com/determined-ai/determined/pull/9113 is merged